doc/changelog.rst

```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :bug:`985` Fix parameterization of used fixtures
```

Comment:
Fix a bug where Slash did not expand parameterizations used through `use_fixture` properly